CMakeLists.txt

```diff
@@ -450,7 +450,7 @@ if(ANDROID)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
-elseif(NOT APPLE AND NOT CMAKE_SYSTEM_NAME MATCHES OpenBSD)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Would it be better to search for librt explicitly, instead of relying on matching SYSTEM_NAME? Or asking CMake "how do I get `clock_gettime` to compile?"?